src/eos-google-chrome-installer.py

```diff
@@ -212,21 +212,23 @@ def _set_as_default_browser(self):
                      'x-scheme-handler/about']
         for mimetype in mimetypes:
             try:
-                subprocess.Popen(['xdg-mime', 'default', 'google-chrome.desktop', mimetype])
-            except OSError:
-                exit_with_error("Couldn't start xdg-mime. Are xdg-utils installed?")
+                subprocess.check_call('xdg-mime default google-chrome.desktop {}'.format(mimetype),
+                                      shell=True)
+            except subprocess.CalledProcessError as e:
+                exit_with_error("Couldn't start xdg-mime: {}".format(str(e)))
         try:
-            subprocess.Popen(['xdg-settings', 'set', 'default-web-browser', 'google-chrome.desktop'])
-        except OSError:
-            exit_with_error("Couldn't start xdg-settings. Are xdg-utils installed?")
+            subprocess.check_call('xdg-settings set default-web-browser google-chrome.desktop',
```

Comment:
The reason of using `shell=True` is not to need to pass the full path to the xdg-settings binary, as this way check_call will search in the PATH.